client/locations-list-map.js

```diff
@@ -952,10 +952,16 @@ function createRequesterListItemEl(entry) {
   entry.domElem = ce('div', 'location');
   const header = ce('div', 'd-flex');
   const headerHospitalInfo = ce('div', 'flex-grow-1');
+  const headerOrgType = ce('div', 'flex-grow-1 d-flex justify-content-end text-pink');
   const headerZoomLink = ce('div', 'icon icon-search entry-zoom-link');
   headerZoomLink.setAttribute('aria-label', 'Zoom to marker');
-  const headerOrgType = ce('div', 'flex-grow-1 d-flex justify-content-end text-pink');
-  ac(headerHospitalInfo, ce('h5', null, [ctn(entry.name), headerZoomLink]));
+  const children = [ctn(entry.name), headerZoomLink];
+  if (document.body.dataset.partnerSite) {
+    const headerPartnerLink = ce('div', 'icon icon-search entry-partner-link'); // TODO replace icon
```

Comment:
If we canonically convert `dataset.partnerSite` into a CSS-class-safe string, then all our partners would have to do is add the CSS for their icon (and add themselves to the whitelist in our routing)